src/admiral/certs/tasks.py

```diff
@@ -17,6 +17,10 @@
     r"([a-z0-9]{2,63}|(?:[a-z0-9][a-z0-9\-]{0,61}[a-z0-9]))\.?$"
 )
 
+# Default timeout values for tasks that perform web requests. See
+# "Warning' at https://docs.celeryq.dev/en/stable/userguide/tasks.html
+CONNECT_TIMEOUT, READ_TIMEOUT = 5.0, 30.0
```

Comment:
We typically don't declare multiple variables on the same line.  I personally think it's easier to read if they are separated, though happy to hear from others if you disagree:
```suggestion
CONNECT_TIMEOUT = 5.0
READ_TIMEOUT = 30.0
```

Reply:
I agree with you. Saving one line isn't worth the trade-off with readability. I'm going to commit you suggestion. Thanks!